modules/ocf_decal/manifests/website.pp

```diff
@@ -0,0 +1,59 @@
+class ocf_decal::website {
+
+  file {
+    ['/srv/www', '/srv/www/decal']:
+      ensure  => directory,
+      owner   => ocfdecal,
+      group   => www-data,
+      require => User['ocfdecal'];
+    ['/srv/ssl', '/srv/ssl/decal']:
+      ensure  => directory,
+      owner   => ocfdecal,
+      group   => www-data,
+      mode    => '0440',
+      require => User['ocfdecal'];
+    '/srv/ssl/decal/decal.key':
+      source  => 'puppet:///private/decal.key',
+      mode    => '0440';
+    '/srv/ssl/decal/decal.crt':
+      source  => 'puppet:///private/decal.crt',
+      mode    => '0440';
```

Comment:
I think you want some dependencies here, like `require => File['/srv/ssl/decal']`, but it's not a big deal in general since these should be executed in order in more recent versions of puppet anyway. It's good to have dependencies here though so that it's clear which other rules/statements are being depended on to execute each of these.